### PR TITLE
D2D-013

### PR DESCRIPTION
**Pull request by Nibaron**
redirect to dashboard page

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+
+  useEffect( () => {
+    router.push('/dashboard');
+  },[router]);
+
   return (
-    <div className=''>
-      This is <strong>Home</strong> page
-    </div>
+     <p>Redirecting...</p>
```
Comment (tajnur007) [inline app/page.tsx:14]: We will not show any text while redirecting, so put here a fragment only `<></>`

File: app/page.tsx [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
 const HomePage = () => {
+  const router = useRouter();
+
+  useEffect( () => {
+    router.push('/dashboard');
+  },[router]);
```
Comment (tajnur007) [inline app/page.tsx:11]: Here we do not need to include the `router` as a dependency, we can keep it empty. So the `useEffect` will looks like as following:
```javascript
useEffect( () => {
  router.push('/dashboard');
} ,[]);
```

**Comment by Nibaron** [comment]
all three changes have been made.

**Review by tajnur007 — APPROVED** [review]
Perfect! Your PR is approved!

**Pull request merged**